SDK does not specify yarn as dependency

**Summary:** 
<!-- Summarize the problem in a few sentences: -->

SDK does not specify yarn as dependency

**Steps to Reproduce:** 
<!-- How can we reproduce the problem? -->

1. Try a release build

<!-- Please attach (or link to) configuration files if applicable -->

**What do you see now?**
<!-- Please attach (or link to) screenshots and logs if applicable -->

```
lorawan-stack  Fetching JS SDK dependencies
bash: ./node_modules/.bin/yarn: No such file or directory
make: *** [sdk.js.deps] Error 127
```

**What do you want to see instead?**
<!-- Please add some examples or mock-ups if applicable -->

Successful build

**How do you propose to implement this?**
<!--
If unsure, add the discussion label and (temporarily) assign the expert
If you cannot assign people, please @mention the experts
-->

https://github.com/rvolosatovs/lorawan-stack/commit/121a09ade77ee00cdd3e96eddb5dda3e3319dde5

**Environment:**
<!-- Your environment: OS/Browser/Gateway/Device/...? Versions? IDs/EUIs? -->

Any

--- a/.make/sdk/main.make
+++ b/.make/sdk/main.make
@@ -21,9 +21,11 @@ sdk.deps: sdk.js.deps
 
 # JS SDK make rules
 
-sdk.js.deps:
+sdk.js.deps: $(YARN)
 	@$(log) "Fetching JS SDK dependencies"
 	@$(YARN_SDK) install $(YARN_FLAGS) --production=false
 
 include .make/sdk/build.make
 include .make/sdk/quality.make
+
+# vim: ft=make

--- a/Makefile
+++ b/Makefile
@@ -47,7 +47,7 @@ docs:
 
 dev-deps: go.deps js.dev-deps
 
-deps: go.deps sdk.deps sdk.js.build js.deps
+deps: go.deps js.deps sdk.deps sdk.js.build
 
 test: go.test js.test sdk.test
 

